server/services/offenderSummaryService.js

```diff
@@ -1,10 +1,20 @@
-const { getOffenderData } = require('../data/oasysClient')
+const { getOffenderData, getSentencePlan } = require('../data/oasysClient')
 const logger = require('../../log.js')
 
 const offenderSummaryData = async (crn, callback) => {
   logger.info(`Calling getOffenderData(${crn})`)
   try {
     const result = await getOffenderData(crn)
+
+    try {
+      if (result.identifiers) {
+        if (result.identifiers.nomisId) {
+          result.sentencePlan = await getSentencePlan('nomisId', result.identifiers.nomisId)
+        }
+      }
+    } catch (err) {
+      logger.info('No sentence plan returned.')
```

Comment:
not great fan of catching all and then not logging anything to say what has gone wrong

Reply:
should it be `error` too?

Reply:
No sentence-plan returned (404) is a valid scenario from the API.

Reply:
🤔 Although logging out the error may have value 👍

Reply:
Is there no way that we can distinguish between a 404 and an actual error calling the service then?